Makefile

```diff
@@ -0,0 +1,43 @@
+.DEFAULT_GOAL := help
+
+WORKING_DIR := chat
+JS_TARGET := $(WORKING_DIR)/public/js/translations
+EXTRACT_DIR := $(WORKING_DIR)/translations/en/LC_MESSAGES
+EXTRACTED_DJANGO := $(EXTRACT_DIR)/django-partial.po
+EXTRACTED_DJANGOJS := $(EXTRACT_DIR)/djangojs-partial.po
+EXTRACTED_TEXT := $(EXTRACT_DIR)/django.po
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+upgrade:
+	pip-compile --output-file test_requirements.txt test_requirements.in
+
+extract_translations: ## extract strings to be translated, outputting .po files
+	cd $(WORKING_DIR) && i18n_tool extract
+	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
+	tail -n +20 $(EXTRACTED_DJANGOJS) >> $(EXTRACTED_TEXT)
```

Comment:
This is currently crashing with:
```
tail: cannot open 'chat/translations/en/LC_MESSAGES/djangojs-partial.po' for reading: No such file or directory
```
Seems no tagged strings are present in static, so this should be removed.

Reply:
Ah, true. I missed the fact it was erroring out since it was creating the `django.po` file. Thanks.